AutomatedTesting/Gem/PythonTests/assetpipeline/asset_processor_tests/bundle_mode_tests.py

```diff
@@ -23,12 +23,11 @@
 @pytest.mark.SUITE_periodic
 @pytest.mark.parametrize('launcher_platform', ['windows_editor'])
 @pytest.mark.parametrize('project', ['AutomatedTesting'])
-@pytest.mark.parametrize('level', ['auto_test'])
+@pytest.mark.parametrize('level', ['TestDependenciesLevel'])
 class TestBundleMode(object):
     def test_bundle_mode_with_levels_mounts_bundles_correctly(self, request, editor, level, launcher_platform,
                                                               asset_processor, workspace, bundler_batch_helper):
-        level_pak = os.path.join("levels", level, "level.pak")
-
+        level_pak = os.path.join("levels", level, "TestDependenciesLevel.spawnable")
```

Comment:
This PR shows the removal of the old `TestDependenciesLevel` level files, but I don't see in the PR the addition of the new `TestDependenciesLevel.spawnable` file, is it missing?

Reply:
I should have likely added it here as well but the new file is in https://github.com/o3de/o3de/pull/6679
It was an older PR I messed up and had to remake. I can add it here as well though PR-6679 should be merged before this one.